### DISCUSSION TITLE
Can we use modules and libraries from Nuxt in Quarkus app with Quinoa?

### DISCUSSION THREAD

**Discussion by xyfr0** [quarkusio/quarkus#43269] [Q&A]
I want to create a web application using tailwind, but when I try to install some libraries and modules. Unfortunately it causes an error. Because of that i separate Nuxt app to diffrent package. Is Tailwind and Pinia supported in Quarkus? because i like to build web app using those

**Comment by phillip-kruger** [comment]
I think you will have to provide more details. Even better an example app that show the error. @ia3andy ^^^

**Reply by ia3andy** [reply]
As Nuxt can be used for Server Side Rendering, it depends on how you use it.. Please could you start a new issue or conversation in Quinoa repo with an example of your issue.
